src/utils/serialization/SchemaHelpers.ts

```diff
@@ -2,14 +2,9 @@ import Ajv from 'ajv'
 
 const ajv = new Ajv()
 
-const dependencyMap: Map<string, string> = new Map()
 const schemaMap: Map<string, object> = new Map()
 const verifyFunctions: Map<string, Ajv.ValidateFunction> = new Map()
 
-export function addSchemaDependency(name: string, requiredBy: string): void {
```

Comment:
Hey @PudgyPug I guess @abdulazeem-tk4vr and you were having discussion on this? Has it reached any conclusion? If so, could you drop a note here? Thanks!

Reply:
I talked with Abdul + Sonali about this. We agreed that it's fine to remove